### ISSUE TITLE
Is cryptography 2.9 supported?

### ISSUE THREAD

**Issue by omz13** [nabla-c0d3/sslyze#428]
**Describe the bug**
`ERROR: sslyze 3.0.1 has requirement cryptography==2.8, but you'll have cryptography 2.9 which is incompatible.`

**To Reproduce**
1. `pipx install yawast` (yes, I am trying to install *yawast* which depends on *sslyze*)
2. See error

**Expected behavior**
*ssylze* installs (with cryptography 2.9)

**Python environment (please complete the following information):**
- macOS Sierra 10.12.6
- Python version: 3.8.2

**Additional context**
I assume this is related to #424 and #425... I suspect the real question is can `sslyze` depend on cryptography 2.9 as well as 2.8

**Comment by weddige** [comment]
This is not an issue of *sslyze*, but *yawast*. It is just not compatible with sslyze 3 and should not install it.

Look at https://github.com/adamcaudill/yawast/blob/master/Pipfile#L16
```
[packages]
yawast = {editable = true,path = "."}
sslyze = "==2.1.4"
requests-mock = "*"
jsonschema = "~=2.6"
validator-collection = "*"
requests = "*"
publicsuffixlist = "*"
dnspython = "*"
urllib3 = "*"
colorama = "*"
nassl = "*"
cryptography = "==2.5"
```

**Comment by weddige** [comment]
And it should not install cryptography 2.9 as well, als it requires cryptography 2.5. Open an issue in their repository, that the yawast package is broken: https://github.com/adamcaudill/yawast/issues

**Comment by nabla-c0d3** [comment]
I've added support for cryptography 2.6 to 2.9 in SSLyze; that's all I can do in regards to this issue.

**Comment by nabla-c0d3** [comment]
Fix released in v3.0.2.
